app/screens/buyers/BuyingView.java

```diff
@@ -2,37 +2,60 @@
 
 import icebase.app.App;
 
+import java.util.ArrayList;
 import java.util.Scanner;
 
 import icebase.app.MenuTitle;
+import icebase.app.Router;
+import icebase.app.enums.CATEGORY_ENUM;
+import icebase.app.enums.SCREEN_ENUM;
 import icebase.app.screens.Screen;
+import icebase.app.types.Store;
 
 public class BuyingView implements Screen {
+    private String storeName;
+    private Store store;
+    private Scanner sc = App.sc;
+    private ArrayList<String> categoryNames = new ArrayList<>();
+    private int choice;
+
+    public BuyingView(Store store) {
+        this.store = store;
+        this.storeName = store.getName();
+        for (CATEGORY_ENUM categoryENUM : CATEGORY_ENUM.values()) {
+            categoryNames.add(categoryENUM.value);
+        }
+        categoryNames.add("return");
```

Comment:
ipa pretty please hehe

`Return` insteand of `return`

Reply:
oks gotchu